chrome/content/ui.js

```diff
@@ -195,6 +196,9 @@ var ui = {
 
   init: function() {
     ui.setPrefs();
+    if (coniks.Enabled) {
+      coniks.init();
```

Comment:
I'd rather just see a call to `coniks.init()` here and then return early in the function body if it isn't enabled.